source/standalone/environments/teleoperation/teleop_se3_agent.py

```diff
@@ -13,7 +13,6 @@
 
 # add argparse arguments
 parser = argparse.ArgumentParser(description="Keyboard teleoperation for Isaac Lab environments.")
```

Comment:
This script already uses `--device` to select the teleoperation device. I believe it has to be renamed in order to avoid conflicts with the changes from this PR.